float64 xPos
float64 yPos
float64 zPos

int8   controlFlag

float64 xPosRef
float64 yPosRef
float64 zPosRef
